packages/x-components/src/x-modules/experience-controls/store/__tests__/actions.spec.ts

```diff
@@ -0,0 +1,89 @@
+import Vuex from 'vuex';
+import { createLocalVue } from '@vue/test-utils';
+import { getMockedAdapter, installNewXPlugin } from '../../../../__tests__/utils';
+import { getExperienceControlsStub } from '../../../../__stubs__/experience-controls-subs.factory';
+import { createExperienceControlsStore, resetExperienceControlsStateWith } from './utils';
+
+describe('testing experience controls module actions', () => {
+  const mockedResponse = getExperienceControlsStub();
+
+  const adapter = getMockedAdapter({
+    experienceControls: mockedResponse
+  });
+
+  const localVue = createLocalVue();
+  localVue.config.productionTip = false; // Silent production console messages.
+  localVue.use(Vuex);
+  const store = createExperienceControlsStore();
```

Comment:
Abstracting the creation of the store like this doesn't make much sense here.
You're creating the `localVue` twice because you need it for the store and for the `installNewXPlugin `